### DISCUSSION TITLE
How to log ticket for ApacheAge support.

### DISCUSSION THREAD

**Discussion by salahuddibitnine** [apache/age#1890] [General]
Greetings,

If I need support using ApacheAge where can I log my ticket to get support?

Regards,
Salahuddin.

**Comment by markgomer** [comment]
Hello @salahuddibitnine , there are some companies that provide support for Apache AGE. 
AGEDB is one of them, please visit their site.
But you can ask any question here on discussions or open an Issue for that.

**Comment by salahuddibitnine** [comment]
Dear Marco,

Thanks for info.

Regards,
Salahuddin.

**Comment by jrgemignani** [comment]
@salahuddibitnine @markgomer Anyone can ask questions or present issues here, about Apache AGE. However, Apache AGE is not affiliated with any other company. Nor can we, as a project, recommend any other company for support. Please take any community members suggestions as just that and at your own risk.
